math/mathcore/src/Delaunay2D.cxx

```diff
@@ -16,9 +16,9 @@
 
 //#include <thread>
 
-// use the triangle library if we do not use CGAL
+// use the CDT library if we do not use CGAL
 #ifndef HAS_CGAL
-#include "triangle.h"
+#include "CDT/CDT.h"
```

Comment:
Olivier, one can add protection to avoid warnings from external code

```
#if !defined(_MSC_VER)
#pragma GCC diagnostic push
#pragma GCC diagnostic ignored "-Wshadow"
#endif

#include "CDT/CDT.h"

#if !defined(_MSC_VER)
#pragma GCC diagnostic pop
#endif
```